integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -523,7 +523,9 @@ def test_concurrent_writer(spark_tmp_path):
 
 @ignore_order
 @pytest.mark.skipif(is_before_spark_320(), reason="is only supported in Spark 320+")
-def test_fallback_to_single_writer_from_concurrent_writer(spark_tmp_path):
+@allow_non_gpu(any=True)
+@pytest.mark.parametrize('enable_aqe', ["true", "false"])
```

Comment:
not a blocker: I'd prefer type-safe Boolean literals `True`, `False`

This is at least the fourth naming for this kind of pytest parameter after `aqe_enabled`, `aqeEnabled` and `adaptive` I see in our code base. Let us standardize on the more common `aqe_enabled` for easier grepping

Reply:
Done.